dkg/node.go

```diff
@@ -69,6 +73,10 @@ func (n *Node) ProcessMessage(msg *types.SSVMessage) error {
 		return n.startNewDKGMsg(signedMsg)
 	case ProtocolMsgType:
 		return n.processDKGMsg(signedMsg)
+	case DepositDataMsgType:
```

Comment:
that should be in the runner struct. 
A node holds multiple runners and a runner should take care of the entire DKG cycle

Reply:
If we don't include this MsgType here. Runner won't be able to get the message. Node and runner are checking the same field.

Reply:
Or should the node check SSVMessage's MsgType instead of SignedMessage's MsgType?

Reply:
Here we only check InitMsgType.